pkg/addons/applier.go

```diff
@@ -59,18 +60,20 @@ type applier struct {
 
 // TemplateData is data available in the addons render template
 type templateData struct {
-	Config                              *kubeoneapi.KubeOneCluster
-	Certificates                        map[string]string
-	Credentials                         map[string]string
-	CredentialsCCM                      map[string]string
-	CCMClusterName                      string
-	CSIMigration                        bool
-	CSIMigrationFeatureGates            string
-	MachineControllerCredentialsEnvVars string
-	RegistryCredentials                 []registryCredentialsContainer
-	InternalImages                      *internalImages
-	Resources                           map[string]string
-	Params                              map[string]string
+	Config                                   *kubeoneapi.KubeOneCluster
+	Certificates                             map[string]string
+	Credentials                              map[string]string
+	CredentialsCCM                           map[string]string
+	CCMClusterName                           string
+	CSIMigration                             bool
+	CSIMigrationFeatureGates                 string
+	MachineControllerCredentialsEnvVars      string
+	AddonOperatingSystemManagerEnabled       bool
```

Comment:
I would drop this Addon prefix.

```suggestion
	OperatingSystemManagerEnabled       bool
```